pkgs/by-name/py/pyprland/package.nix

```diff
@@ -26,7 +26,6 @@ python3Packages.buildPythonApplication rec {
     "pyprland.common"
     "pyprland.plugins"
     "pyprland.plugins.interface"
-    "pyprland.plugins.ironbar"
```

Comment:
Why aren't the rest of the plug-ins in this list as well? (I barely have any experience with python so it's possible that this question might be stupid)

Reply:
Because of this https://github.com/hyprland-community/pyprland/blob/main/pyprland/ipc.py#L14
It executes on `import`. But there is no `HYPRLAND_INSTANCE_SIGNATURE` when building package

Reply:
Thanks for explaining.